build.sh

```diff
@@ -422,6 +409,7 @@ case "$COMMAND" in
         ;;
 
     "ios-swift")
+        sh build.sh ios-dynamic
```

Comment:
Building RealmSwift.framework takes care of building Realm.framework. This step is only necessary to copy the built framework to the expected location in `build/ios`.